src/System.Text.Encoding.CodePages/tests/EncodingCodePages.cs

```diff
@@ -633,7 +633,7 @@ public static void TestEncodingDisplayNames(int codePage, string webName, string
 
         // This test is run as part of the default mappings test, since it modifies global state which that test
         // depends on.
-        public static void TestRegister1252()
+        private static void TestRegister1252()
```

Comment:
I wonder if the comment meant "is not run".  But I'm not sure what heuristic was applied for private vs Fact(Skip) vs comment out vs delete.

Reply:
The comment means that is invoked by a method which is marked as fact as the parent method depends on global state manipulation.